### ISSUE TITLE
The health circle around mother spaceship is not correctly centered

### ISSUE THREAD

**Issue by ketanhwr** [ketanhwr/reflexio#36]
Can be fixed easily since the coordinates need to be modified by a bit.

**Comment by Hn5624** [comment]
Fixed it.  Also
There seems to be a lot of duplicate functions on the levels.js.
Can i have permission to clean out some more of the unused functions? Game works fine without them, maybe they are code people were trying to rewrite but never finished.
Once I read and understand some more of the code, I will add some light documentation to them so future users will be able to read it easier.

**Comment by ketanhwr** [comment]
@Hn5624 check this: https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by flyer2403** [comment]
Has this issue been solved?

**Comment by ketanhwr** [comment]
@flyer2403 it's still open, so yes.

**Comment by Hn5624** [comment]
I fixed it and submitted the changes, you can load the game and you should be able to see if its fixed, basically the healthbar around your spacestation was off-centered.

If its still not fixed that means the creator didn't accept my changes or hasnt responded.


________________________________

**Comment by ketanhwr** [comment]
@Hn5624 I didn't receive any pull request whatsoever.

**Comment by flyer2403** [comment]
@Hn5624 could you please make a pull request?

**Comment by p-shannon** [comment]
@Hn5624 Oooooh, Okay, I know what you did. You cloned the repo instead of forking it. When you're contributing to a project you don't own, you need to make a fork of it so you that you have ownership (of a fork). Changes you make in your fork can reflected in the main repo (we'll call this upstream for now on) by making a pull request.

Now, a fix should be easy enough I believe; 

1. Make a new repository that has the same name as this one; leave it empty, it'll be filled in step three.
2. Change the origin of the repository you worked on to match the one you made; something like `git remote set-url origin https://github.com/hn562/reflexio.git`
3. From the repo that you worked on push to origin `git push origin`; At this point you've populated your fork with the work you've done on the cloned upstream.
4. Now go to this repo from the web browser and click pull requests; then make a new pull request
5. Make sure the base says 'ketanhwr/master' and the compare says 'hn5624/master'
6. Lastly fill out the PR with the information suggested by the contributions guide and wait for it to be merged

Hope you're able to follow these instructions as I imagined you following them in my head, this too is my first contribution to an open source project. I'll be fairly active on github if you have and questions.

**Comment by Hn5624** [comment]
Yeah lol, github was extremely confusing to me at first, it makes much more sense now.

I think CMD prompts may be easier to use than the actual github desktop app.


________________________________

**Comment by p-shannon** [comment]
Yeah I agree with that. Definitely do yourself a favor and get used to the command line interface from now on. Best of luck in open sourced world!

**Comment by dilsheen** [comment]
Is this still up for grabs?

**Comment by sans-clue** [comment]
Is this still up for grabs?
